dependencies.props

```diff
@@ -131,16 +130,29 @@
     <XUnitPerformanceApiDependency Include="xunit.performance.execution" />
     <XUnitPerformanceApiDependency Include="xunit.performance.metrics" />
     <StaticDependency Include="@(XUnitPerformanceApiDependency)">
-      <Version>$(XUnitPerformanceApiVersion)</Version>
+      <Version>$(XunitPerformanceApiPackageVersion)</Version>
     </StaticDependency>
 
     <StaticDependency Include="xunit.console.netcore">
-      <Version>$(XUnitConsoleNetCoreVersion)</Version>
+      <Version>$(XunitConsoleNetcorePackageVersion)</Version>
     </StaticDependency>
 
     <DependencyBuildInfo Include="@(StaticDependency)">
       <PackageId>%(Identity)</PackageId>
       <UpdateStableVersions>true</UpdateStableVersions>
     </DependencyBuildInfo>
   </ItemGroup>
+
+  <!-- Override isolated build dependency versions with versions from Repo API. -->
+  <Import Project="$(DotNetPackageVersionPropsPath)"
+          Condition="'$(DotNetPackageVersionPropsPath)' != ''" />
+
+  <!--
+    Map PackageVersion properties that don't match the Repo API naming conventions. This must be
+    defined after the DotNetPackageVersionPropsPath Import so overrides apply correctly.
+  -->
+  <PropertyGroup>
+    <!-- A single property used to get all CoreCLR packages in various places. -->
+    <CoreClrPackageVersion>$(MicrosoftNETCoreRuntimeCoreCLRPackageVersion)</CoreClrPackageVersion>    
```

Comment:
Would it be better if we just changed all the places to use the new name?  We do this for Platforms because BuildTools needs it to be a specific name. But here I think we can just update the repo.

Reply:
This is just a port of @dagood's change but I agree I will look to see if I can eliminate this.